README.md

```diff
@@ -24,33 +26,14 @@
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
```

Comment:
Your readme file has too many empty sections, I suggest you could either fill in those sections with information about your project or simply remove them in order to make your Readme file more professional.